@here/harp-mapview/lib/Utils.ts

```diff
@@ -240,6 +258,23 @@ export namespace MapViewUtils {
         mapView.update();
     }
 
+    /**
+     * The function doing a pan when [[MapView]]'s active [[ProjectionType]] is spherical.
+     *
+     * @param mapView MapView instance.
+     * @param fromWorld Start vector representing the scene position of a geolocation.
+     * @param toWorld End vector representing the scene position of a geolocation.
+     */
+    export function rotateGlobe(
```

Comment:
rotateGlobe is actually rotating the camera about the globe, to make it look like the globe is rotating (as far as I can tell)

Why not just rotate the globe and keep the camera still?

Reply:
Another comment, rotating the camera is also fine, I would just rename the function to mention that the camera is moved, as opposed to the globe, something like rotateCameraAroundGlobe... or equivalent

Reply:
Ah indeed. All the camera logic was not implemented by me so I don't know the ins and outs of the decision, but I will rename to this for clarity.